src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs

```diff
@@ -38,60 +38,55 @@
-                             (stringify_map (m "inputs"))
-                             (stringify_map (m "outputs"))
-                             (stringify_map (m "prerequisites"))])
-                          (:method-confs props))})])])})
+        [table/AdvancedTable
+         (let [header (fn [children] [:div {:style {:fontWeight 600 :fontSize 13
+                                               :padding "14px 16px"
+                                               :borderLeft "1px solid #777777"
+                                               :color "#fff" :backgroundColor (:header-darkgray style/colors)}}
+                                      children])
+               cell (fn [children] [:span {:style {:paddingLeft 16}} children])]
+           {:columns [[:div {:style {:padding "13px 0px 12px 12px" :backgroundColor (:header-darkgray style/colors)}}
+                       [:input {:type "checkbox" :ref "allcheck"}]]
+                      (header "Name")
+                      (header "Namespace")
+                      (header "Type")
+                      (header "Workspace Name")
+                      (header "Method")
+                      (header "Config")
+                      (header "Inputs")
+                      (header "Outputs")
+                      (header "Prerequisites")]
+            :column-widths [42 200 200 100 150 200 220 200 200 200]
+            :data (:method-confs props)
+            :row-props (fn [row-num conf]
+                         {:style {:fontSize "80%" :fontWeight 500 :lineHeight 1.2
+                                  :paddingTop 10 :paddingBottom 7
+                                  :backgroundColor (if (even? row-num) (:background-gray style/colors) "#fff")}})
+            :render-cell (fn [row-num col-num conf]
+                           (case col-num
+                             0 [:div {:style {:paddingLeft 12}} [:input {:type "checkbox"}]]
+                             1 (cell [:a {:href "javascript:;"
+                                          :style {:color (:button-blue style/colors)
+                                                  :textDecoration "none"}} (conf "name")])
+                             2 (cell (conf "namespace"))
+                             3 (cell (conf "root-ent-type"))
+                             4 (cell (str ((conf "workspaceName") "namespace") ":"
+                                       ((conf "workspaceName") "name")))
+                             5 (cell (str ((conf "methodStoreMethod") "methodNamespace") ":"
+                                       ((conf "methodStoreMethod") "methodName") ":"
+                                       ((conf "methodStoreMethod") "methodVersion")))
+                             6 (cell (str (get (conf "methodStoreConfig") "namespace") ":"
+                                       ((conf "methodStoreConfig") "name") ":"
+                                       ((conf "methodStoreConfig") "version")))
```

Comment:
The fields of methodStoreConfig are as follows, so these aren't showing up in the UI:
```
    "methodStoreConfig": {
      "methodConfigNamespace": "repo_method_configs",
      "methodConfigName": "repo_method_config1",
      "methodConfigVersion": "2"
    }
```